prismicio/slice-machine
# Add multi-tab for code in SetupModal

[Linear ticket](https://linear.app/prismic/issue/SM-917/aauser-i-can-see-multitab-to-code-block-in-setupmodal-in-slice)

- Add tabs for npm and yarn in Setup modal
- Optimize tabs to prevent unnecessary reload

To test it:
- Remove prop `localSimulatorURL` from `sm.json`
- Access the simulator
- Check the first Setup tab
- alternate between yarn and npm: both code and copy should match
- the other code blocks should still match previous design

How it looks
<img width="607" alt="Capture d’écran 2023-01-03 à 17 29 00" src="https://user-images.githubusercontent.com/6568065/210537393-5ab45b67-b4d3-4755-80ad-bc96bd0cf7fc.png">
